oslo/torch/_C/__init__.py

```diff
@@ -147,3 +147,11 @@ def nvcc_args(self, maxrregcount: int = None):
             additional_flags.append(f"-maxrregcount={maxrregcount}")
 
         return nvcc_flags + additional_flags
+
+class CUDABinder(Binder):
+    @property
+    def name(self):
+        return "cuda"
```

Comment:
Let's change this name to `oslo`.
it will be saved in the `.cache/torch_extension/NAME`. cuda is ambiguous.

Reply:
And I think the name `cuda` looks ambiguous.
Let's change this `oslo-softmax`, `oslo-adam`, .. etc.
Does this make sense?

Reply:
Totally agree with you! I'll tell you if I come up with other idea

Reply:
Thanks !